test/consts.test.ts

```diff
@@ -17,6 +27,30 @@ describe('consts', () => {
         });
     });
 
+    describe('PROFILE_NAME', () => {
+        it('REGEX works as expected', () => {
+            // Valid cases
+            expect(PROFILE_NAME.REGEX.test('John Doe')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('Anonymous')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('John123')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('John-Doe')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('Org_Example')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test(':/JohnDoe')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test(':/a/Simple.Name')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('John:/Doe/')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('Simple:.//Name')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('Joh////:/n-Doe')).toBe(true);
+            expect(PROFILE_NAME.REGEX.test('user:name')).toBe(true);
+
+            // Invalid cases
+            expect(PROFILE_NAME.REGEX.test('user@name')).toBe(false);
+            expect(PROFILE_NAME.REGEX.test('user>name')).toBe(false);
+            expect(PROFILE_NAME.REGEX.test('user<name')).toBe(false);
+            expect(PROFILE_NAME.REGEX.test('example://test')).toBe(false);
+            expect(PROFILE_NAME.REGEX.test('example://////test')).toBe(false);
```

Comment:
Nit: I'd include examples that showcase why we want such regex. `<a href="https://malicious.site">Safe John Doe</a>`